processor/resourcedetectionprocessor/internal/heroku/internal/metadata/testdata/config.yaml

```diff
@@ -5,8 +5,6 @@ all_set:
       enabled: true
     heroku.app.id:
       enabled: true
-    heroku.app.name:
```

Comment:
I know this was broken behavior, but would it make sense to transition out this option since removing it is a breaking change?

Reply:
Like keeping it as no-op with a warning for a couple of versions? It'll be just a few manual changes to support that. Resource attributes don't yet have this `if_enabled_set` stuff. I'm actually working towards that when I found this issue. I don't believe anyone applied this since it was introduced only last version. You you and @TylerHelmuth think that it worth the effort, I can apply the suggestion

Reply:
If it was only applied in the last release, you're right that it's unlikely to be used, so I'm okay breaking it. The closest thing I can find that would document what we should do here is the [stability guarantees for scraping receivers](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#beta), which only says we SHOULD follow the transition process for beta components.